components/ai_chat/common/buildflags/buildflags.gni

```diff
@@ -9,6 +9,6 @@ import("//build/config/ui.gni")
 declare_args() {
   enable_ai_chat =
       (brave_channel == "nightly" || brave_channel == "development") &&
-      (is_win || is_mac || is_linux || is_android)
+      (is_win || is_mac || is_linux || is_android || is_ios)
```

Comment:
Should we just remove is_`os` conditions altogether?

Reply:
seems like it!

Reply:
Fixed